content/posts/rust_interview_questions.md

```diff
@@ -144,14 +144,16 @@ is also `'static`.
 
 
 The `'static` lifetime is required due to how threads work. The spawned thread might
-outlive the calling call so it needs to own all the data.
+outlive the calling scope or thread, so it may only access unowned data which is 
```

Comment:
I like your direction, but I think it still needs some polish: 

`T: 'static` means that `T` can be either owned ie `String` or `Vec` but also it can be a borrow for a lifetime `'static`: ie `&'static str`

So maybe the second part of the sentence should read: `so it may not access data that has been invalidated by some other code.`

What do you think?